test/web_test.rb

```diff
@@ -39,16 +40,17 @@
         differential_temperature_upper_deg_c: '26.7',
         wind_speed_metres_per_second: '2.8',
         wind_direction_deg_true_north: '169.3',
-        sigma_deg_true_north: '32.2'
+        sigma_deg_true_north: '32.2',
+        original_reading_datetime_string: '20 March 2018 11:00:00 AM AEDT'
       )
     end
 
     it 'should supply a CSV of all of the data' do
       get '/csv'
       last_response.body.must_equal <<~CSV
-        id,location_name,scraped_at,latest_reading_recorded_at,pm2_5_concentration_ug_per_m3,pm10_concentration_ug_per_m3,co_concentration_ppm,no2_concentration_ppm,differential_temperature_lower_deg_c,differential_temperature_upper_deg_c,wind_speed_metres_per_second,wind_direction_deg_true_north,sigma_deg_true_north
-        2,Allen St AQM,2018-03-20 01:03:28 UTC,20 March 2018 11:00:00 AM AEDT,12.0,48.4,0.1,0.009,27.1,26.7,2.8,169.3,32.2
-        1,Haberfield Public School AQM,2018-03-20 01:03:32 UTC,20 March 2018 11:00:00 AM AEDT,17.0,44.1,0.07,0.006,26.9,25.8,3.0,175.3,31.3
+        id,location_name,scraped_at,latest_reading_recorded_at,pm2_5_concentration_ug_per_m3,pm10_concentration_ug_per_m3,co_concentration_ppm,no2_concentration_ppm,differential_temperature_lower_deg_c,differential_temperature_upper_deg_c,wind_speed_metres_per_second,wind_direction_deg_true_north,sigma_deg_true_north,original_reading_datetime_string
+        2,Allen St AQM,2018-03-20 01:03:28 UTC,20 March 2018 11:00:00 AM AEDT,12.0,48.4,0.1,0.009,27.1,26.7,2.8,169.3,32.2,20 March 2018 11:00:00 AM AEDT
```

Comment:
It would be good to have raw date and the correctly recorded date side-by-side in the data exported so it's clear they're related. However that might be more code so maybe an improvement for another day?

Reply:
> However that might be more code so maybe an improvement for another day?

It might be nice to more `scraped_at` down to the end too, as it's a bit more of a meta field than the data people are interested in. I think we leave this for another day though, just to keep this focused and moving.